stan/math/opencl/multiply.hpp

```diff
@@ -37,6 +37,28 @@ inline auto multiply(const matrix_cl& A, const matrix_cl& B) {
     temp.zeros();
     return temp;
   }
+  if (A.rows() == 1 && triangular_view_A == TriangularViewCL::Entire
+      && triangular_view_B == TriangularViewCL::Entire) {
+    try {
+      opencl_kernels::row_vector_matrix_multiply(
+          cl::NDRange(temp.cols() * 64), cl::NDRange(64), A.buffer(),
```

Comment:
This 64 should be retrieved from the implementations parameters like we read the local sizes:
`int local = opencl_kernels::matrix_multiply.make_functor.get_opts().at("THREAD_BLOCK_SIZE");`